Stripe/PublicHeaders/STPAddCardViewController.h

```diff
@@ -50,11 +50,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, strong, nullable) STPUserInformation *prefilledInformation;
 
-/**
- If you're using the token generated from STPAddCardViewController to make a Managed Account, you should set this property to the currency that account will use. Otherwise, you should leave it empty. For more information, see https://stripe.com/docs/api#create_card_token-card-currency
- */
-@property (nonatomic, copy, nullable) NSString *managedAccountCurrency;
```

Comment:
I don't really understand this use case. Do you know how user's who were using token+managedAccountCurrency previously will migrate?

Reply:
I talked to @danwang-stripe - users who are migrating should get the same functionality[0] for free with PaymentMethods.  I've called this out `CHANGELOG.md` but maybe it should be in `MIGRATING.md`.

[0] I'm not familiar with what it does beyond https://stripe.com/docs/api/tokens/create_card#create_card_token-card-currency

Reply:
👍